src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int SIZE = 12;
+    private static final int INITIAL_VALUE = -1;
+    private static final int FAILURE = -1;
+    private static final int MAX_SIZE = 12;
```

Comment:
This constant is related to SIZE. Changing only one leads to an error (inconsistency).